packages/react-ui/src/Titles/Title/index.js

```diff
@@ -23,12 +24,19 @@ export const Title = ({
     shift={shift}
     {...props}
   >
-    <StyledTitle leftStripped={!topStripped} as={as} shift={shift}>
-      <Stripe
-        rounded={variant !== "primary"}
-        variant={variant}
-        {...(!topStripped && { position: "left", length: "100%" })}
-      />
+    <StyledTitle
+      unstriped={unstriped}
```

Comment:
I wonder if Title should have a prop stripped where possible values none | top | left  (default to none)
As reading the code it seems weird to have a `leftStripped = !topStripped`

Reply:
Let's give it a try !